config/initializers/ministerial_report.rb

```diff
@@ -204,6 +204,16 @@ def self.determine_dominant_course_subject_for_report(course_name, course_level,
       end
     end
 
+    # take the first subject value if unable to match above
+    if !subject
+      logger = Logger.new(STDOUT)
+      logger.debug 'SUBJECTS:'
+      logger.debug(subject_names_and_codes)
+      logger.debug(subject_names)
+      logger.debug(subject_names.first)
```

Comment:
If this logging is needed longer-term can we just use `Rails.logger` so that it ends up in the normal log, whatever that is? Would it be useful to write a more literal message if it's needed, like _Falling back to #{subject_names.first} for..._?

Reply:
Hi @stevehook I'm logging this because I'm currently having problems with the build. All tests are passing locally but for some reason when the build. For some reason it flips the subjects which causes the test to fail (I'm struggling to replicate this behaviour locally)
![image](https://user-images.githubusercontent.com/47917431/142176645-4cf8fa3c-26f9-48b9-9720-d452d29dde6f.png)
(The subject returned should be 'business studies')